.github/workflows/update.yml

```diff
@@ -16,9 +16,9 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
I think the unit test errors are because of node upgrade

Reply:
i tried to stack all needed updates, so i will do it step by step later

Reply:
ok there is problem with @types/jest since Node 19 https://github.com/sinonjs/fake-timers/issues/463 so downgrade of Nodejs in matrix check shows and confirms that.

I will try to upgrade to latest 29.5.x and see